docs/source/faqs.rst

```diff
@@ -70,6 +70,40 @@ This would result in the local "Sync Demo" project being copied to the
 specified S3 bucket. Under-the-hood, it uses the AWS CLI ``sync`` method. So,
 you'd need to have the AWS CLI installed and ensure your credentials are set.
 
+Why does Rubicon offer a Prefect integration?
+=============================================
+
+`prefect <https://docs.prefect.io/>`_ is a popular workflow management system
```

Comment:
maybe mention how the prefect tasks are just wrappers around existing rubicon functions so implementing additional workflow units of work for other tools should be trivial